### ISSUE TITLE
Convert guide to Markdown

### ISSUE THREAD

**Issue by ericholscher** [writethedocs/www#1509]
We should use the same format for the guide. It's confusing to have rst & markdown mixed, and now Sphinx has great markdown support:

* Convert form recommonmark to [myst](https://myst-parser.readthedocs.io/en/latest/)
* https://github.com/executablebooks/rst-to-myst

**Comment by ericholscher** [comment]
We are moving to a more recent version of markdown here: #1738

**Comment by djwfyi** [comment]
So is the idea to convert all existing content to markdown?

**Comment by ericholscher** [comment]
@djwfyi Not really. Mostly just to offer the option to write in RST or MD, whichever is easiest. I think we should try to keep various sections in a similar format if possible though, so it's less confusing.

This was about the guide specifically, with the goal of making contribution easier. However, as you've noted elsewhere, the guide is still not super well structured. I think probably some kind of merging of the guide & topic index is likely a better approach in terms of content management, and that's outside of any specific tooling or markup choice.

**Comment by rukundob451** [comment]
@ericholscher, is this issue still open to be worked on?
